src/Microsoft.Framework.Configuration/Internal/ConfigurationFocus.cs

```diff
@@ -72,5 +73,10 @@ public void Set(string key, string value)
         {
             return _root.GetConfigurationSections(_prefix + key);
         }
+
+        public void Reload()
+        {
+            throw new NotImplementedException();
```

Comment:
I don't like the idea of not doing anything since it gives the user wrong impression that some operation happened and it succeeded. Isn't it ok to throw with a good error message?May be I can do an `InvalidOperationException` with error message like `Reload can only be called on top-level configuration`. I mean just for one method having 2 interfaces(one that has `Reload` and other that does not) seems  like an overkill, unless we have more of such methods (which we don't).